ui/admin/app/templates/scopes/scope/auth-methods/index.hbs

```diff
@@ -187,16 +187,12 @@
               }}
             {{/if}}
           </message.description>
-          <message.link
-            @route='scopes.scope.auth-methods.new'
-            @query={{hash type='password'}}
-          >
-            <Rose::Icon @name='flight-icons/plus-circle-16' />
-            {{t 'titles.new'}}
-          </message.link>
           {{#if (can 'create collection' this.scope collection='auth-methods')}}
-            <message.link @route='scopes.scope.auth-method.new'>
-              <Rose::Icon @name='plus-circle-outline' />
+            <message.link
+              @route='scopes.scope.auth-methods.new'
+              @query={{hash type='password'}}
```

Comment:
How does this handle oidc auth method creation? It looks like clicking on new auth method link will automatically navigate to password auth method form.

Reply:
Handled in [PR](https://github.com/hashicorp/boundary-ui/pull/746). 👍